network-api/networkapi/settings.py

```diff
@@ -408,6 +408,7 @@
 
 LOCALE_PATHS = (
     os.path.join(BASE_DIR, 'locale'),
+    os.path.join(BASE_DIR, 'networkapi/buyersguide/templates/about/locale'),
```

Comment:
Just realized that files in this folder were correctly extracted, but translations were NOT picked up by Django.
Looks like we need to specify that path, as it’s not the default `[app name]/locale`